Lib/ufo2ft/kernFeatureWriter.py

```diff
@@ -9,27 +12,34 @@ class KernFeatureWriter(AbstractFeatureWriter):
     Uses the kerning rules contained in an RFont's kerning attribute, as well as
     glyph classes from parsed OTF text. Class-based rules are set based on the
     existing rules for their key glyphs.
+
+    Uses class attributes to match UFO glyph group names and feature syntax
+    glyph class names as kerning classes, which can be overridden.
     """
 
+    leftUfoGroupRe = r"^public\.kern1\.(.+)"
+    rightUfoGroupRe = r"^public\.kern2\.(.+)"
+    leftFeaClassRe = r"@_(.+)_L"
```

Comment:
The widely-accepted convention of ufo2, that of MetricsMachine, is basically `@MMK_L_` sth, see e.g. https://github.com/adobe-fonts/source-sans-pro/blob/master/Roman/Regular/kern.fea#L7. 
Is the syntax that you have here that of Glyphs, somehow?